db/seeds.d/020-provisioning_templates_list.rb

```diff
@@ -93,14 +109,19 @@ def seeded_templates
       ]
     end
 
-    def kinds
-      kinds = {}
+    @@kinds ||= begin
```

Comment:
Please keep the `kinds` method to prevent that huge change of the hash above. What I was suggesting was actually Ruby memoization design pattern: https://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/ - simply just use `@kinds` as in the examples and keep using the method please.

Reply:
Sure, got it.